README.markdown

```diff
@@ -60,6 +60,12 @@ Add these lines to your `~/.bashrc`, `~/.profile`, or `~/.zshrc` file to have it
 For manual upgrade with `git`, change to the `$NVM_DIR`, pull down the latest changes, and check out the latest version:
 
     cd "$NVM_DIR" && git pull origin master && git checkout `git describe --abbrev=0 --tags`
+    
+After upgrading, reload the new version:
+
+    nvm unload && source nvm.sh
```

Comment:
there's not actually a need to `nvm unload` though - just freshly sourcing `nvm.sh` should do it. Also, this should use `.` instead of `source` since it's more portable.